src/AxesUtil/AxesUtil.js

```diff
@@ -97,11 +97,19 @@ class AxesUtil {
     }
 
     const x = axisFunc(scale)
-      .ticks(ticks)
-      .tickValues(tickValues)
-      .tickFormat(tickFormatter)
-      .tickSize(config.tickSize)
-      .tickPadding(config.tickPadding);
+      .tickFormat(tickFormatter);
+    if (ticks) {
+      x.ticks(ticks);
+    }
+    if (tickValues) {
+      x.tickValues(tickValues);
+    }
+    if (config.tickSize) {
+      x.tickSize(config.tickSize);
+    }
+    if (config.tickPadding) {
```

Comment:
This will falsely skip a valid user configured `tickPadding` of `0`; these probably should all explicitly test against `undefined`

Reply:
Nice catch, thanks!